packages/ag-charts-enterprise/src/series/nightingale/nightingaleSeries.ts

```diff
@@ -2,12 +2,18 @@ import { _ModuleSupport, _Scale, _Scene, _Util } from 'ag-charts-community';
 
 import { RadialColumnSeriesBase } from '../radial-column/radialColumnSeriesBase';
 import type { RadialColumnNodeDatum } from '../radial-column/radialColumnSeriesBase';
+import { prepareRadialColumnAnimationFunctions, resetRadialColumnSelectionFn } from '../radial-column/radialColumnUtil';
 
-const { Sector } = _Scene;
+const { seriesLabelFadeInAnimation } = _ModuleSupport;
+const { Sector, motion } = _Scene;
 
 export class NightingaleSeries extends RadialColumnSeriesBase<_Scene.Sector> {
     static className = 'NightingaleSeries';
 
+    constructor(moduleCtx: _ModuleSupport.ModuleContext) {
+        super(moduleCtx, { animationResetFns: { item: resetRadialColumnSelectionFn } });
```

Comment:
Why is `resetRadialColumnSelectionFn` not a class method to be overriden by sub-classes?

Reply:
Ultimately we need to move towards a composition approach rather than class-hierarchy approach for series behaviours - for animation I'm attempting to use re-usable functions for this purpose.